.github/code_review_template.md

```diff
@@ -0,0 +1,37 @@
+# Review
+
+## Comments
+
+<!-- If you have any comments for the PR author, write them here. -->
+
+## Checklist 
+
+<!-- If you are not sure about any of the following items, assign an additional reviewer to cover those points. -->
+
+### General
+
+- [ ] The PR provides a clear high-level description for a general audience, including photos or GIFs if applicable
+- [ ] The PR includes a detailed change list
+- [ ] The PR is associated with a bug, user story, or other work item on Azure DevOps
+- [ ] Changes to existing functionality have been communicated to affected users (including other DataHub team members), or there is a plan to communicate to affected users, or not applicable
+
+### Code Quality
+
+- [ ] The code is easily understandable and maintainable
+- [ ] The code is consistent with .NET standards and the project architecture
+- [ ] The code is adequately documented (such as in-code comments or dev documentation)
+- [ ] The code does not contain duplicate code that could be abstracted
+
+### Tests
+
+- [ ] New code behaves as expected and achieves its intended purpose
+- [ ] Existing functionality behaves as expected and has been tested
+- [ ] The code has been tested in both English and French, if applicable
+- [ ] The PR includes adequate unit tests to cover code changes
+- [ ] The PR does not break existing tests
+
+### Additional Requirements
+
+- [ ] The code does not introduce any security vulnerabilities
+- [ ] The PR includes updated localization files or makes no changes that impact localization
+- [ ] The PR follows accessibility requirements or makes no changes that impact accessibility
```

Comment:
I really like these and think they deserve a bit more details or points on their own. We could also add some points around the UI/UX and the style guide

Reply:
I've added a commit that expands on localization a bit to cover what devs have to do for i18n.

I think something similar would be good for a11y to check things like headings, proper component usage, alt text, etc. Maybe @davidreneuw has a list already lol